physionet-django/console/views.py

```diff
@@ -1493,3 +1522,32 @@ def get_queryset(self):
             qs = qs.filter(username__icontains=self.q)
 
         return qs
+
+@login_required
+@user_passes_test(is_admin, redirect_field_name='project_home')
+def known_referencess(request):
```

Comment:
is the extra "s" in `known_referencess` intentional? it's used in the view and the template name.

Reply:
Fixed in next commit :)